ceph.spec.in

```diff
@@ -446,6 +444,7 @@ Requires:       ceph-base = %{_epoch_prefix}%{version}-%{release}
 Requires:       python%{_python_buildid}-bcrypt
 Requires:       python%{_python_buildid}-pecan
 Requires:       python%{_python_buildid}-six
+Requires:       python%{_python_buildid}-requests
```

Comment:
please move this line before `Requires:       python%{_python_buildid}-six` so this block of `Requires` are ordered alphabetically?